.travis.yml

```diff
@@ -75,19 +85,23 @@ jobs:
 
     - <<: *smoke
       node_js: '12'
+      name: 'Node.js v12'
 
     - <<: *smoke
       node_js: '10'
+      name: 'Node.js v10'
 
     - stage: precache
       script: true
+      name: 'Prime cache'
 
 notifications:
   email: false
-  urls:
-    # for gitter mochajs/mocha
-    - secure: fUrHenYJs+pTuLtgBRoYyrlyfVekxaIGmLWq7bhUUqBj/7p5eCkQFn13LlPht0/4WWZOiPBcdTN7tKnz3Ho7ATUJhAchvOWDUgL5gtTvOzeCHbPuCvHz/VLK6hMoPdbLA45M864NDLotfHvyh62WgQaVw9iPc80eb+umaDPrYiU=
-    # for gitter mochajs/contributors
-    - secure: rGMGYWBaZgEa9i997jJHKzjI8WxECqLi6BqsMhvstDq9EeTeXkZFVfz4r6G3Xugsk3tFwb/pDpiYo1OK36kA5arUJTCia51u4Wn+c7lHKcpef/vXztoyucvw6/jXdVm/FQz1jztYYbqdyAOWC2BV8gYvg5F8TpK05UGCe5R0bRA=
-  on_success: change
-  on_failure: always
+  webhooks:
+    urls:
+      # for gitter mochajs/mocha
```

Comment:
IMO we don't need this for mochajs/mocha, it doesn't seem to work anyway in this room.
In mochajs/contributors it works.

Reply:
k

Reply:
removed the main gitter room webhook